### PR TITLE
COMMERCE-5477 Money Order Configuration Message does not save

### PR DESCRIPTION
**Pull request by luca-pellizzon**

### CODE REVIEW

**Comment by luca-pellizzon** [comment]
ci:test:sf

**Comment by luca-pellizzon** [comment]
ci:test:relevant

**Comment by luca-pellizzon** [comment]
ci:test:sf

**Comment by luca-pellizzon** [comment]
ci:test:relevant

File: modules/apps/commerce/commerce-payment-method-money-order/src/main/java/com/liferay/commerce/payment/method/money/order/internal/portlet/action/EditMoneyOrderCommercePaymentMethodConfigurationMVCActionCommand.java [hunk 1]
```diff
@@ -80,13 +82,26 @@ private void _updateCommercePaymentMethod(ActionRequest actionRequest)
...
+
+		Map<String, String> messageMap = new HashMap<>();
 
 		for (Map.Entry<String, String> entry : unicodeProperties.entrySet()) {
-			modifiableSettings.setValue(entry.getKey(), entry.getValue());
+			messageMap.put(entry.getKey(), entry.getValue());
 		}
 
+		String messageAsLocalizedXML = LocalizationUtil.getXml(
+			messageMap, "", "messageAsLocalizedXML");
```
Comment (dsitu) [inline modules/apps/commerce/commerce-payment-method-money-order/src/main/java/com/liferay/commerce/payment/method/money/order/internal/portlet/action/EditMoneyOrderCommercePaymentMethodConfigurationMVCActionCommand.java:100]: @luca-pellizzon we can use `StringPool.BLANK` here.
```suggestion
			messageMap, StringPool.BLANK, "messageAsLocalizedXML");
```

File: modules/apps/commerce/commerce-lang/src/main/resources/content/Language.properties [hunk 2]
```diff
@@ -880,6 +880,7 @@ mercanet=Mercanet
 mercanet-configuration-help=Mercanet accepts only EUR. The primary currency of the store should therefore be EUR.
 mercanet-description=Pay via Mercanet.
 merchant-id=Merchant ID
+message-as-localized-xml=Message as localized XML
```
Comment (dsitu) [inline modules/apps/commerce/commerce-lang/src/main/resources/content/Language.properties:883]: lets capitalize it.
```suggestion
message-as-localized-xml=Message as Localized XML
```

**Review by dsitu — CHANGES_REQUESTED** [review]

**Comment by luca-pellizzon** [comment]
ci:test:sf

**Comment by luca-pellizzon** [comment]
ci:test:relevant

**Comment by luca-pellizzon** [comment]
ci:test:relevant

**Review by dsitu — APPROVED** [review]

**Comment by riccardo-alberti** [comment]
ci:forward

**Comment by luca-pellizzon** [comment]
ci:test:sf

**Comment by luca-pellizzon** [comment]
ci:test:sf

**Comment by luca-pellizzon** [comment]
ci:test:relevant

**Comment by riccardo-alberti** [comment]
ci:forward

**Pull request closed without merging**